src/manifolds/Stiefel.jl

```diff
@@ -421,8 +420,7 @@ tangent space at $q=\operatorname{retr}_p(d)$ using the [`CaleyRetraction`](@ref
 vector_transport_direction(M::Stiefel, p, X, d, ::CaleyVectorTransport)
 
 function vector_transport_direction!(::Stiefel, Y, p, X, d, ::CaleyVectorTransport)
-    Pp = p * p'
-    Pp .= one(Pp) - 1 // 2 * Pp
+    Pp = I - p * p'
```

Comment:
Didn't you want `Pp = I - 1 // 2 * p * p'`? This changes the behavior of code you replaced.

Reply:
Oops. Yes, the formulae in the papers read with 1/2, though that surprises me a little, I'll add those.